src/sidebar/store/modules/drafts.js

```diff
@@ -97,8 +98,6 @@ function createDraft(annotation, changes) {
  * An empty draft has no text and no reference tags.
  */
 function deleteNewAndEmptyDrafts() {
-  const { removeAnnotations } = require('./annotations');
-
```

Comment:
Heh, this is an odd duck.

Reply:
I believe this was done to avoid a circular dependency between this module and `./annotations.js`. Circular dependencies are allowed between ES modules, although best avoided if possible for other reasons.